chirpstack/luci-app-chirpstack-concentratord-target-rak7268v2/htdocs/luci-static/resources/view/chirpstack/concentratord-slot1.js

```diff
@@ -5,21 +5,25 @@
 return view.extend({
     render: function () {
         const options = {
+            hideFields: {
+                usb: true,
+                gnss: false
```

Comment:
I'm not sure if this gateway will ever have a GNSS module? If that assumption is correct, maybe we can hide this flag too?

Reply:
Initially it comes without GPS. Now, wouldn't it be necessary to get the right time on a relay gateway, as an example? If so, we should offer the user the option to say if it has or not a GPS.